packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -72,7 +72,7 @@ export interface BaseServiceProps {
    *
    * @default - AWS Cloud Map service discovery is not enabled.
    */
-  readonly serviceDiscoveryOptions?: ServiceDiscoveryOptions;
+  readonly awsCloudMapOptions?: AWSCloudMapOptions;
```

Comment:
Remove the aws prefix from both prop and type names. We also usually prefer to avoid the “options” postfix from the prop name (it’s good on the type), so this should be:

```ts
cloudMap: CloudMapOptions
```